src/initializeApp.js

```diff
@@ -27,6 +27,9 @@ function setOptions(urlParams, store) {
   if (urlParams.insecure === `true`) {
     store.commit(`setInsecureMode`)
   }
+  if (urlParams.production === `true`) {
```

Comment:
remove production mode switch?

Reply:
But then we cannot test the `/create` endpoint in development. I mean this bit:

![image](https://user-images.githubusercontent.com/40721795/80403411-d3b1d700-88bf-11ea-88f9-361e1bb68bc2.png)

Reply:
In doubt you can just manually change the value in code. This is a very rare case

Reply:
OK, then I will remove it